jbt-it/neomdb
# 459 server trainee applicant profile

How to test:
- add sufficient data to the db
- Check the route /application/:id to get the details of an applicant
- Check the route /application/:id/image to get the image of an applicant
- Run the tests
